tests/ert/unit_tests/ensemble_evaluator/test_ensemble_evaluator.py

```diff
@@ -672,3 +673,54 @@ async def test_snapshot_on_resubmit_is_cleared(evaluator_to_use):
             assert snapshot.get_fm_step("0", "1")["status"] == FORWARD_MODEL_STATE_INIT
 
         await monitor.signal_done()
+
+
+@pytest.mark.integration_test
+async def test_signal_cancel_does_not_cause_evaluator_dispatcher_communication_to_hang(
```

Comment:
Tbh, I like to test setup but missing a self-explanatory assert (as a punchline :smile: )